### DISCUSSION TITLE
What is NeoForge?

### DISCUSSION THREAD

**Discussion by TheBrambleShark** [neoforged/NeoForge#184] [Ideas]
I found NeoForge on the Exaroton site when picking software for a new server, but after some research there doesn't seem to be anything to tell me what NeoForge actually is.

I have read that it is a fork of Forge, but even after reading your news article I'm still left with questions. I think the following questions should be answered in short form directly on the webpage, perhaps linking back to the long-form article for people who are curious about the drama. Follow-up questions in each bullet point are rhetorical and are intended to help guide some of the answers.

1. Why was it forked?
2. How is NeoForge different from Forge? Is it technical? Is it philosophical? Why should I be interested in the project?
3. Who runs the project? Is this Forge 2.0 from the same developers? Is it a divestment from the original project?
4. What is the target audience of this project? Primarily people who are interested in Forge? Are you interested in pulling people away from Sponge or Fabric/Quilt?

Currently the homepage seems to be targeted entirely towards people who have a firm understanding of the project and its goals, are interested in using this project on their server, but need help actually using the software. This is a good first step, but including information to help onboard people who are not in this category will be very beneficial moving forward.

**Comment by TelepathicGrunt** [comment]
Don't forget to read the blog on the bottom of the Neoforge official page https://neoforged.net/news/theproject/

**Reply by TheBrambleShark** [reply]
This is the "long-form article" I refer to. A couple of the questions are answered there, specifically 1 and 3, but the point of this post is to get all of this onto the home page for people discovering the project for the first time. We always have people who are trying out server hosting for the first time or for who have not hosted in several years and are just discovering new server software for the first time.

Also presenting viewers with a giant wall of text is a good way to get them to click away, so this is why I specifically mention answering these "in short form directly on the webpage."
